src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -189,10 +113,7 @@ export const StakingOpportunities = ({ assetId }: StakingOpportunitiesProps) =>
         Cell: ({ value }: { value: string }) => {
           return hasActiveStakingOpportunities ? (
             <Amount.Crypto
-              value={bnOrZero(value)
-                .div(`1e+${asset.precision}`)
-                .decimalPlaces(asset.precision)
-                .toString()}
+              value={bnOrZero(value).toString()}
```

Comment:
`BN`ing a string to then stringify it? 🤔 
Note: it should already come to the component as a stringified number so if the goal is to zero it, that shouldn't be a concern

Reply:
Oops ! some leftover ... fixed !

Reply:
done !